modules/animation/include/modules/animation/animationcontroller.h

```diff
@@ -95,12 +95,10 @@ class IVW_MODULE_ANIMATION_API AnimationController : public AnimationControllerO
     /// Returns the current state of the controller, whether it is playing, or pausing, and such.
     const AnimationState& getState() const;
 
-    /// Returns playback mode such as loop or swing and such.
-    const AnimationPlaySettings& getPlaybackSettings() const { return settingsPlay_; }
-    AnimationPlaySettings& getPlaybackSettings() { return settingsPlay_; }
-    void setPlaybackSettings(const AnimationPlaySettings& newSettings);
```

Comment:
Interesting. We did not use that? I assume that looping and swinging still work, though, right?

Reply:
I guess we intended to use it but never did. Probably better to redo it if we need it.
Everything still works, it was completely unused :)